### ISSUE TITLE
RancherOs does not build with Docker 17.06-ce

### ISSUE THREAD

**Issue by bjfr** [rancher/os#2002]
**RancherOS Version: (ros os version)**
v1.0.3

**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)**
Tested to build on Osx and Ubuntu 16.04

When building with Docker 16.04, I get the following error:

```
Step 5/12 : ENV VERSION 2.0.14
 ---> Using cache
 ---> 0b157fef0af8
Step 6/12 : ADD https://www.kernel.org/pub/linux/utils/kernel/kexec/kexec-tools-$VERSION.tar.gz .
Downloading [==================================================>] 398.7 kB/398.7 kB
Error processing tar file(exit status 1): link /kexec-tools-2.0.14/purgatory/arch/i386/entry32-16.S /kexec-tools-2.0.14/purgatory/arch/i386/entry32-16.S: no such file or directory
FATA[0001] exit status 1
FATA[0175] exit status 1
Reaping losing child 0x7fa5f4a006f0 PID 8750
make: *** [default] Error 1
Removing child 0x7fa5f4a006f0 PID 8750 from chain.
````

Apparently it does not manage to untar the kexec-tool-1.0.14.tar.gz. Reverting to Docker 17.03-2, the build finishes without problems!

**Comment by SvenDowideit** [comment]
oh gee.

**Comment by superseb** [comment]
Maybe related? https://github.com/moby/moby/issues/33849

**Comment by SvenDowideit** [comment]
I've backported the change from `ADD` to `RUN wget` - so this should be solved in both the `master` and `v1.0.x` branches. - @superseb, you're right - and I guess that means its fixed in Docker 17.06.01-ce

yay for regressions - closing - thank you all :)
